easybuild/easyconfigs/c/CUDAcore/CUDAcore-11.1.1.eb

```
@@ -0,0 +1,18 @@
+easyblock = "EB_CUDA"
+name = 'CUDAcore'
+version = '11.1.1'
+local_nv_version = '455.32.00'
+
+homepage = 'https://developer.nvidia.com/cuda-toolkit'
+description = """CUDA (formerly Compute Unified Device Architecture) is a parallel
+ computing platform and programming model created by NVIDIA and implemented by the
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://developer.download.nvidia.com/compute/cuda/%(version)s/local_installers/']
+sources = ['cuda_%%(version)s_%s_linux%%(cudaarch)s.run' % local_nv_version]
+checksums = ['3eae6727086024925ebbcef3e9a45ad379d8490768fd00f9c2d8b6fd9cd8dd8f']
```

This should be extended, to also cover the checksum for the `ppc64le` installer, like we're doing for CUDA 11.0:

```suggestion
checksums = [
    {
        'cuda_%%(version)s_%s_linux.run' % local_nv_version:
            '3eae6727086024925ebbcef3e9a45ad379d8490768fd00f9c2d8b6fd9cd8dd8f',
        'cuda_%%(version)s_%s_linux_ppc64le.run' % local_nv_version:
            '023e571fe26ee829c98138dfc305a92279854aac7d184d255fd58c06c6af3c17',
    }
]
```